BUILD.gn

```diff
@@ -0,0 +1,23 @@
+# Copyright 2018 The Dawn Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import("//third_party/dawn/scripts/dawn_overrides_with_defaults.gni")
+
+group("all") {
+  testonly = true
+  deps = [ "src/tests:webnn_tests" ]
+  if (webnn_standalone) {
```

Comment:
Where does `webnn_standalone` come from? It shouldn't exist in `//third_party/dawn/scripts/dawn_overrides_with_defaults.gni`

Reply:
It might be clearer to import `build_overrides/webnn.gni` directly.

Reply:
fixed in the new commit https://github.com/webmachinelearning/webnn-native/pull/1/commits/ecb095846e319d3bebcfb2520f1f6ab25e0a8e3d